app/controllers/githubs_controller.rb

```diff
@@ -0,0 +1,13 @@
+class GithubsController < ApplicationController
+  def create
+    response = request.env['omniauth.auth']
+    current_user.update(token: github_params(response))
+    redirect_to dashboard_path
+  end
+
+  private
+
+  def github_params(response)
+    response["credentials"]["token"]
+  end
```

Comment:
Regarding the github_params and line 4. I wrote something similar with tutorials create
`tutorial = Tutorial.create(tutorial_info_params(tutorial_info))`
``` 
def tutorial_info_params(tutorial)
    { title: tutorial.title,
      description: tutorial.description,
      thumbnail: tutorial.thumbnail,
      playlist_id: tutorial.id }
  end
```

Reply:
So you want this syntax for the params?